pkg/chunk/encoding/bigchunk.go

```diff
@@ -131,16 +128,15 @@ func (b *bigchunk) UnmarshalFromBuf(buf []byte) error {
 			return err
 		}
 
-		var start, end int64
-		start, end, reuseIter, err = firstAndLastTimes(chunk, reuseIter)
+		var start int64
+		start, reuseIter, err = firstTime(chunk, reuseIter)
```

Comment:
Will this function now need to populate `b.end` somewhere?

Reply:
That seems a good shout. 
And a missing test.

Reply:
It turns out `end` is completely unnecessary - it is only used to shortcut the case where `FindAtOrAfter()` is called with a target past the end of the chunk, and this never happens because we have from/through times on each chunk at a higher level.

I extended the unit tests to fail on the previous code.

Reply:
:+1: Thanks for looking into that!